### PR TITLE
Allow dynamic placement of copyright component

### LINKED ISSUE #552: Dynamic positioning of psammead-copyright component
**Is your feature request related to a problem? Please describe.**
The position of the `psammead-copyright` component will vary per service, depending on whether the script is right-to-left or left-to-right, e.g.
- news: bottom left of image

Screen Shot 2019-06-07 at 11 11 11

- persian: bottom right of image
Screen Shot 2019-06-07 at 11 11 26

Therefore, we need to pass a prop to the component that determines its position within its parent container.

Default case can be the 'left' value. 

**Describe the solution you'd like**
By default the copyright component will be placed bottom-left of its parent container. However, I would like to pass a prop that can override this position. This prop should be named `position` which defaults to `left` or `right`. There should be new stories added to storybook to demonstrate this dynamic positioning.

**Testing notes**
Check Storybook examples. 

Dev insight: Visual regression, unit tests.

### PR DESCRIPTION
**Pull request by 12**
Resolves #552 

**Overall change:** Change the default absolute position of `psammead-copyright` from right, to left. Allow it to be overridden to `right` with a prop named `position`.

**Code changes:**

- Snapshots updated
- Pass optional prop `position` to `psammead-copyright`.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-copyright/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.4.0   | [PR#617](https://github.com/bbc/psammead/pull/617) Add optional `position` prop, and default position to left of its parent container |
```
Comment (sareh) [inline packages/components/psammead-copyright/CHANGELOG.md]: Since we're using this in production, we should really bump this to be v1.0.0

Comment (12) [reply packages/components/psammead-copyright/CHANGELOG.md]: Gotcha

**Review by pjlee11 — APPROVED** [review]
Nice one 👍

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**